lib/nerves_hub_web/channels/device_channel.ex

```diff
@@ -612,4 +615,6 @@ defmodule NervesHubWeb.DeviceChannel do
       0
     end
   end
+
+  defp is_safe_to_request_extensions?(version), do: Version.match?(version, ">= 2.5.2")
```

Comment:
```suggestion
  defp is_safe_to_request_extensions?(version), do: Version.match?(version, "> 2.5.2")
```
`2.5.2` is the currently released NHL version, but the extensions work has not been released to hex. So we probably want that as non-inclusive greater-than to capture the next version which should have the extensions work